wagtail_footnotes/test/tests/test_blocks.py

```diff
@@ -1,23 +1,121 @@
+import json
+
 from django.test import TestCase
 from wagtail import VERSION as WAGTAIL_VERSION
 
 from wagtail_footnotes.blocks import RichTextBlockWithFootnotes
+from wagtail_footnotes.models import Footnote
+from wagtail_footnotes.test.models import TestPageStreamField
 
 if WAGTAIL_VERSION >= (3, 0):
     from wagtail import blocks
     from wagtail.fields import StreamBlock
+    from wagtail.models import Page
 else:
     from wagtail.core import blocks
     from wagtail.core.fields import StreamBlock
+    from wagtail.core.models import Page
 
 
 class TestBlocks(TestCase):
-    def test_custom_block(self):
+    def setUp(self):
```

Comment:
suggestion: use `setupTestData` for this. https://adamj.eu/tech/2021/02/10/new-testing-features-in-django-3.2/#setuptestdata-isolation